app/code/Magento/Reports/Model/ResourceModel/Order/Collection.php

```diff
@@ -814,19 +816,27 @@ public function addSumAvgTotals($storeId = 0)
      * @param string $baseSubtotalRefunded
      * @param string $baseSubtotalCanceled
      * @param string $baseDiscountCanceled
+     * @param string $baseDiscountRefunded
      * @return string
      */
     protected function getTotalsExpression(
         $storeId,
         $baseSubtotalRefunded,
         $baseSubtotalCanceled,
-        $baseDiscountCanceled
+        $baseDiscountCanceled,
+        $baseDiscountRefunded = '0'
```

Comment:
Unfortunately, we are not allowed to change public and protected methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
Hi @sidolov 

I have observed that you have initialized "$baseDiscountRefunded = '0'" in the method signature. So is this the change you are talking about and already did yourself? or you would like me to remove the additional added parameter to function's signature completely?